package ru.practicum.shareit.user.exceptions;

public record UserErrorResponse(String error) {
}